buildSrc/src/main/groovy/airbyte-bulk-connector.gradle

```diff
@@ -156,14 +171,32 @@ class AirbyteBulkConnectorPlugin implements Plugin<Project> {
             dependsOn project.tasks.matching { it.name ==~ /(compile|spotbugs)[a-zA-Z]*Java/ }
         }
 
-        project.tasks.register('integrationTestJava', Test) {
-            description = 'Runs the integration tests.'
+        // This task exists solely so that we can run the "modern" integration tests
+        // in non-docker mode on our laptops.
+        // It's not a dependency of any other task.
+        project.tasks.register('integrationTestNonDocker', Test) {
+            description = 'Runs the integration tests in non-docker mode.'
             group = 'verification'
-            testClassesDirs = project.sourceSets.integrationTestJava.output.classesDirs
-            classpath = project.sourceSets.integrationTestJava.runtimeClasspath
+            testClassesDirs = project.sourceSets.integrationTest.output.classesDirs
+            classpath = project.sourceSets.integrationTest.runtimeClasspath
             useJUnitPlatform()
         }
 
+        // For historical reasons (i.e. airbyte-ci), this task is called integrationTestJava.
+        // airbyte-ci only wants to run a task with exactly that name.
```

Comment:
I can fix this. File an issue and assign it to me.

Reply:
https://github.com/airbytehq/airbyte-internal-issues/issues/10260